packages/backend/.env.example

```diff
@@ -1,13 +1,15 @@
 DATABASE_URL="postgresql://postgres:postgres@localhost:5432/vsebude_db?schema=public"
 PORT=3001
 
+# Email envs
+EMAIL_SERVICE_API_KEY=
+EMAIL_SERVICE_ADDRESS=
```

Comment:
Why you added `EMAIL_SERVICE_ADDRESS`? You don't use it. Is it required for SDK?